### PR TITLE
Adding Jira integration documentation

### PR DESCRIPTION
**Pull request by anais-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/jira-step/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/jira-step/docs/

File: _docs/integrations/jira-integration.md [hunk 1]
```diff
@@ -7,7 +7,32 @@ redirect_from:
...
+## Using your own jira-cli
+
+Alternatively, you can use your own jira-cli by adding the following steps to your Dockerfile:
+
+{% highlight yaml %}
+FROM python:2-alpine
+RUN apk add -U gcc musl-dev linux-headers openssl-dev libffi-dev && pip install jira-cli
+{% endhighlight %}
+
+And then running the Dockerfile.
```
Comment (kostis-codefresh) [inline _docs/integrations/jira-integration.md:38]: Add a "what to read next" on this page.

File: _docs/yaml-examples/examples/sending-the-notification-to-jira.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+And the comment, including the URL to the pipeline, should be added to your Jira issue:
+
+{% include image.html
+lightbox="true"
+file="/images/integrations/jira/jira-comment.png"
+url="/images/integrations/jira/jira-comment.png"
+alt="Comment in Jira"
+max-width="50%"
+%}
```
Comment (kostis-codefresh) [inline _docs/yaml-examples/examples/sending-the-notification-to-jira.md:82]: Add a "what to read next" section

File: _data/nav.yml [hunk 3]
```diff
@@ -290,6 +290,8 @@
       sub-pages:
         - title: Sending the notification to Slack
           url: "/sending-the-notification-to-slack"
+        - title: Sending the notification to Jira
+          url: "/sending-the-notification-to-jira"
```
Comment (kostis-codefresh) [inline _data/nav.yml:294]: You should also add the link here https://codefresh.io/docs/docs/yaml-examples/examples/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/jira-step/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/jira-step/docs/

**Pull request merged**